cvmfs/cvmfs.cc

```diff
@@ -2105,15 +2104,24 @@ static int Init(const loader::LoaderExports *loader_exports) {
   cvmfs::chunk_tables_ = new ChunkTables();
 
   // Runtime counters
-  atomic_init64(&cvmfs::num_fs_open_);
-  atomic_init64(&cvmfs::num_fs_dir_open_);
-  atomic_init64(&cvmfs::num_fs_lookup_);
-  atomic_init64(&cvmfs::num_fs_lookup_negative_);
-  atomic_init64(&cvmfs::num_fs_stat_);
-  atomic_init64(&cvmfs::num_fs_read_);
-  atomic_init64(&cvmfs::num_fs_readlink_);
-  atomic_init64(&cvmfs::num_fs_forget_);
-  atomic_init32(&cvmfs::num_io_error_);
+  cvmfs::num_fs_open_ = cvmfs::statistics_->Register("cvmfs.no_fs_open",
+      "Number of opened files");
+  cvmfs::num_fs_dir_open_ = cvmfs::statistics_->Register("cvmfs.no_fs_dir_open",
+      "Number of opened directories");
+  cvmfs::num_fs_lookup_ = cvmfs::statistics_->Register("cvmfs.no_fs_loopup",
```

Comment:
That would be `n_fs_lookup`, same for the other counters below (ever growing counters)